draft-ietf-httpbis-digest-headers.md

```diff
@@ -1221,6 +1221,10 @@ Content-Type: application/problem+json
 ~~~
 {: title="Response advertising the supported algorithms"}
 
+Note that a 5xx response would have been legitimate too,
+since this specification does not prescribe any behavior
+on status codes.
```

Comment:
How about we move this to the end of first paragraph of this section? E.g. at L1196 we can say something like 

"Alternatively a server can also reject
the request and return a response with
error status code such as 4xx or 5xx. 
This specification does not prescribe
any requirement on status code selection;
the follow example illustrates one possible
option."

Reply:
Ok